### PR TITLE
Feature/eslint settings

### PR DESCRIPTION
**Pull request by kirillstepanchuk**
configured eslint and rewrote code with eslint rules

### CODE REVIEW

File: app-backend/package.json [hunk 1]
```diff
@@ -5,23 +5,30 @@
...
 	},
 	"author": "Kirill Stepanchuk <kirill.steapnchuk.96@mail.ru>",
 	"license": "ISC",
 	"dependencies": {
-		"airbnb": "0.0.2",
 		"cookie-parser": "^1.4.5",
 		"cors": "^2.8.5",
 		"dotenv": "^8.2.0",
-		"eslint": "^7.25.0",
+		"eslint-config-airbnb": "^19.0.2",
```
Comment (vik-buchinski) [inline app-backend/package.json]: please, read about  dependencies and devDependencies and fix deps in your package.json

File: app-frontend/package.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --env mode=dev --open --hot",
-    "build": "webpack --env mode=prod",
+    "start": "webpack-dev-server --config ./webpacks/webpack.dev.js --open --hot",
```
Comment (vik-buchinski) [inline app-frontend/package.json:8]: 👍  for the '--config ./webpacks/webpack.dev.js '

File: app-frontend/package.json [hunk 3]
```diff
@@ -56,9 +53,14 @@
     "react-redux": "^7.2.6",
     "react-router-dom": "^5.2.0",
     "redux": "^4.1.2",
+    "redux-devtools-extension": "^2.13.9",
     "redux-saga": "^1.1.3",
     "regenerator-runtime": "^0.13.9",
     "styled-components": "^5.3.3",
-    "webpack-bundle-analyzer": "^4.5.0"
+    "terser-webpack-plugin": "^5.2.5",
```
Comment (vik-buchinski) [inline app-frontend/package.json]: the same comment about deps here

File: app-frontend/src/components/App/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+// export { default } from './App';
```
Comment (vik-buchinski) [inline app-frontend/src/components/App/index.js]: barrel file with just a comment?
You shouldn't commit commented out code at all

File: app-frontend/src/store/sagas/worker.js [hunk 5]
```diff
@@ -10,10 +10,7 @@ export function* loadWorkersData(action) {
     const { payload } = action;
 
     const responseData = yield call(() => axios.get(
-      `${API_URL}/workers?pageNumber=${payload.page}
-			&positionType=${payload.filterParameters.positionType}
-			&sortingType=${payload.filterParameters.sortingType}
-			&time=${payload.filterParameters.time}`,
+      `${API_URL}/workers?pageNumber=${payload.page}&positionType=${payload.filterParameters.positionType}&sortingType=${payload.filterParameters.sortingType}&time=${payload.filterParameters.time}`,
```
Comment (vik-buchinski) [inline app-frontend/src/store/sagas/worker.js]: here is the way how to fix issue with long string:
https://github.com/axios/axios#browser

just build query params in the appropriate way.

File: app-frontend/webpacks/webpack.common.js [hunk 6]
```diff
@@ -40,8 +40,8 @@ module.exports = (env) => {
       }),
     ],
     // eslint: {
-    //     configFile: './.eslintrc.js',
```
Comment (vik-buchinski) [inline app-frontend/webpacks/webpack.common.js:43]: what the reason for committing commented out code here?

**Review by vik-buchinski — CHANGES_REQUESTED** [review]

**Review by vik-buchinski — APPROVED** [review]

**Comment by kirillstepanchuk** [comment]
configured eslint and rewrote code with eslint rules

**Pull request merged**